Common subdirectories: ros_controllers-release-release-noetic-diff_drive_controller-0.18.0-1/cfg and ros_controllers-release-release-noetic-diff_drive_controller-0.18.0-1.boost-headers/cfg
diff --color -u ros_controllers-release-release-noetic-diff_drive_controller-0.18.0-1/CMakeLists.txt ros_controllers-release-release-noetic-diff_drive_controller-0.18.0-1.boost-headers/CMakeLists.txt
--- ros_controllers-release-release-noetic-diff_drive_controller-0.18.0-1/CMakeLists.txt	2020-10-11 09:55:26.000000000 +0200
+++ ros_controllers-release-release-noetic-diff_drive_controller-0.18.0-1.boost-headers/CMakeLists.txt	2020-11-02 17:40:35.119139438 +0100
@@ -20,7 +20,7 @@
     urdf
 )
 
-find_package(Boost REQUIRED COMPONENTS headers)
+find_package(Boost REQUIRED)
 
 # Declare a catkin package
 generate_dynamic_reconfigure_options(cfg/DiffDriveController.cfg)
Common subdirectories: ros_controllers-release-release-noetic-diff_drive_controller-0.18.0-1/include and ros_controllers-release-release-noetic-diff_drive_controller-0.18.0-1.boost-headers/include
Common subdirectories: ros_controllers-release-release-noetic-diff_drive_controller-0.18.0-1/src and ros_controllers-release-release-noetic-diff_drive_controller-0.18.0-1.boost-headers/src
Common subdirectories: ros_controllers-release-release-noetic-diff_drive_controller-0.18.0-1/test and ros_controllers-release-release-noetic-diff_drive_controller-0.18.0-1.boost-headers/test
